### PR TITLE
ZWEB-3674 JF Make All Library Packages Fixed & Fix Security Issues

### PR DESCRIPTION
**Pull request by Jacob-Friesen**
**Replication**
1. Run the install and audit: `npm install && npm audit`
1. Run the tests: `npm test`
1. Run the linting: `npm run eslint`
1. Confirm that all the gulp tasks still run especially `gulp build` which generates the libs.

Note a lot of unneeded functionality like karma PhantomJS or automatic JS file wrapping is no longer needed, so it was removed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,42 +8,36 @@
...
+    "karma": "5.0.5",
+    "karma-babel-preprocessor": "6.0.1",
     "karma-chrome-launcher": "2.2.0",
-    "karma-phantomjs-launcher": "^0.2.3",
-    "karma-wrap-preprocessor": "^0.1.0",
-    "phantomjs": "^1.9.19",
-    "phantomjs-polyfill": "0.0.2",
+    "karma-jasmine": "0.3.6",
+    "karma-jshint": "0.1.0",
     "zerv-ng-core": "z-open/zerv-ng-core#1.1.3",
```
Comment (etelmon) [inline package.json:37]: ng core is out-dated... should it not be 1.1.8?

File: package.json [hunk 2]
```diff
@@ -8,42 +8,36 @@
   },
   "version": "1.2.10",
```
Comment (etelmon) [inline package.json:9]: 1.2.11?

File: package.json (hunk 1 above)
Comment (Jacob-Friesen) [reply package.json]: That was only release about 1h ago. So now it has been updated. This repository has no READY flag on PRs unlike our main one. I have now updated.

File: package.json (hunk 2 above)
Comment (Jacob-Friesen) [reply package.json]: I added that now too. I was too focused on the other changes needed here.

File: package.json [hunk 3]
```diff
@@ -6,44 +6,38 @@
...
-    "gulp-ng-annotate": "^1.1.0",
-    "gulp-watch": "^4.3.5",
-    "jasmine-core": "^2.4.1",
-    "karma": "^0.13.19",
-    "karma-babel-preprocessor": "^6.0.1",
-    "karma-jasmine": "^0.3.6",
-    "karma-jshint": "^0.1.0",
+    "angular-mocks": "1.5.0",
+    "babel-preset-env": "1.6.0",
+    "babel-preset-es2015": "6.22.0",
```
Comment (etelmon) [inline package.json:17]: those dependencies are very different from the ones in zerv-ng-core:

  "devDependencies": {
    "@babel/core": "7.7.7",
    "@babel/preset-env": "7.7.7",
    "angular-mocks": "1.7.9",
    "babel-preset-env": "1.7.0",
    "eslint": "6.8.0",
    "eslint-config-google": "0.14.0",
    "gulp": "4.0.2",
    "gulp-babel": "8.0.0",
    "gulp-clean": "0.4.0",
    "gulp-concat": "2.6.0",
    "gulp-iife": "0.4.0",
    "gulp-ng-annotate": "2.1.0",
    "jasmine-core": "3.5.0",
    "karma": "5.0.8",
    "karma-babel-preprocessor": "8.0.1",
    "karma-chrome-launcher": "3.1.0",
    "karma-jasmine": "3.0.1",
    "karma-jshint": "0.1.0"
  },

Comment (Jacob-Friesen) [reply package.json:17]: Those dependencies are just for dev testing and building, so they will have have an affect on the main application. I just took the latest develop (with that recent branch merge) and fixed package version and upgraded for security when needed.

Comment (etelmon) [reply package.json:17]: Yes, it is no major but just inconsistent with other zerv ng libs.

**Pull request merged**